mitodl/open-discussions
# new design for course card

#### Pre-Flight checklist


#### What are the relevant tickets?

part of #1997

#### What's this PR do?

This PR implements a new `CourseCard` component for displaying courses and uses it on the course home page (`/courses`). It should match what's showed in the updated designs (see on invision) with the exception of the 'favorite' button which isn't implemented yet.

#### How should this be manually tested?

check out the course home page and make sure things look good. we'll change over the course search result page to use this card for display search results in a separate PR.

#### Screenshots (if appropriate)

![featuredwow](https://user-images.githubusercontent.com/6207644/60984622-04615980-a30a-11e9-9400-a64c34fc75df.png)
